src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props

```diff
@@ -22,7 +22,7 @@
       <Version>1.1.8</Version>
     </NETCoreApp11Dependency>
     <NETCoreApp11Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.1.8</Version>
+      <Version>1.1.9-servicing-26607-01</Version>
```

Comment:
Lets not do this change yet. We should only do this once we ship the 1.1 stable, we don't want to tie the shipping of these 2 release together.

Reply:
I'm confused here, why build CoreClr if we don't want Core-Setup to consume it for this release?

Reply:
Actually never mind, I get what you're saying. Will remove